### ISSUE TITLE
pre-render error for the `[agent].tsx`

### ISSUE THREAD

**Issue by realtouseef** [realtouseef/valopedia#10]
Apparently getting pre-render error for the `[agent].tsx`

A bit about the errors: The API has 2 `Sova`, one with `isPlayableCharacter = true` and the other one with `false`, and we are fetching the one with `true` because it has all the data available. 

Error is occuring because:
- NextJS is creating static pages for every agent and the `Sova with false` is being returned but with no data -- something like that.
- running fine on local but when pushed to vercel or yarn build, the error occurs.

![image](https://user-images.githubusercontent.com/81577619/193435107-89a61616-c220-47d9-80f9-ed34c4d42bed.png)

**Comment by jengkhaw95** [comment]
Hi @realtouseef, it seems like the error can be solved by filtering out the 'unwanted' Sova on getStaticProps (Before passing to getStaticPaths). 
Please let me know if this is open for contribution. Thanks

**Comment by realtouseef** [comment]
@jengkhaw95 Yeah, I feel like the same. And yes, it is open for contribution, please go ahead if you would like to contribute.
